.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+
+python:
+ - "3.5"
+
```

Comment:
I'm not sure what you mean. What I plotted is already the data with signal to noise ratio = 16, as explained repeatedly in code's comments, in this pull request's comment, and in the README file. I also included a density plot with all data with no SN cutoff  just for comparison.

Also, I'm curious why does it need to use python 2 test while 3.5 is working?